community/github/our-repositories.md

```diff
@@ -59,7 +84,23 @@ Update the website contributors guide with general CI and specific Jekyll
 information.
 :::
 
+## [pyOpenSci Sphinx Theme](https://github.com/pyOpenSci/pyos-sphinx-theme)
+
+**Platform:** Sphinx book template that builds on top of the pydata_sphinx_theme
+
+All of our pyOpenSci sphinx books (Handbook, packaging guide, software review guide ) have been customized to match our pyOpenSci branding. This repo contains the start of a sphinx theme that will incorporate all of our branding so we do not have to manually apply the branding and update the branding individually in each repo. Rather we can update branding in the theme and it will be applied across all of our repositories that use the theme.
```

Comment:
```suggestion
All of our pyOpenSci Sphinx books (handbook, packaging guide, software review guide ) have been customized to match our pyOpenSci branding. This repo contains the start of a Sphinx theme that will incorporate all of our branding, so we do not have to manually apply the branding and update it individually in each repo. Instead, we can update branding in the theme and it will be applied across all of our repositories that use the theme.
```